tests/integration/s2n_handshake_test_s_client.py

```diff
@@ -548,7 +548,7 @@ def elliptic_curve_test(host, port, fips_mode):
     Acceptance test for supported elliptic curves. Tests all possible supported curves with unsupported curves mixed in
     for noise.
     """
-    supported_curves = ["P-256", "P-384"]
+    supported_curves = ["P-256", "P-384", "X25519"]
```

Comment:
I think we'll need to make this conditional on the `libcrypto` parameter to this script. our integration tests that build s2n with openssl 1.0.2 won't be able to negotiate x25519